src/core/api/index.js

```diff
@@ -0,0 +1,20 @@
+import { Schema, arrayOf, normalize } from 'normalizr';
+
+// FIXME: Load this from something.
+const API_HOST = 'https://addons-dev.allizom.org';
```

Comment:
It was giving me `undefined`. That file uses `process.env` throughout so I don't know how it would work in the browser.